server/sequencer_manager_test.go

```diff
@@ -190,7 +201,12 @@ func TestSequencerManagerSingleLogNoSigner(t *testing.T) {
 	mockStorage := storage.NewMockLogStorage(mockCtrl)
 	mockSf := keys.NewMockSignerFactory(mockCtrl)
 
-	mockSf.EXPECT().NewSigner(gomock.Any(), stestonly.LogTree.PrivateKey).Return(nil, errors.New("no signer for this tree"))
+	var keyProto ptypes.DynamicAny
+	if err := ptypes.UnmarshalAny(stestonly.LogTree.PrivateKey, &keyProto); err != nil {
+		t.Errorf("Failed to unmarshal stestonly.LogTree.PrivateKey: %v", err)
+	}
+
+	mockSf.EXPECT().NewSigner(gomock.Any(), keyProto.Message).Return(nil, errors.New("no signer for this tree"))
```

Comment:
In this case, it doesn't seem that we care about the keyProto passed to the signer. Maybe remove the unmarshal and use `gomock.Any()` for keyProto?